index.markdown

```diff
@@ -292,23 +292,23 @@ In next sections, we will see stack opcodes that receive parameters from stack.
 
 ### `pick` (opcode `0x79`)
 
-`pick` reads `n` and copies `n`-th element to top stack. Example:
+`pick` reads and drops the number at the top of the stack, which we will call `n`, and copies the `n`-th element to the top of the stack. Example:
```

Comment:
Good idea.. I think that changing to `pick consumes n` could be even simpler.

Reply:
The issue that I came across when following the tutorial is that `n` is not defined anywhere so you have to end up guessing what it is by looking at the examples. I was trying to fix that with these changes.

Reply:
Perhaps: `consumes top-stack value n`...

Reply:
That's a great idea, I'll change it!